stable/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -22,6 +25,9 @@ spec:
         component: "{{ .Values.controller.name }}"
         release: {{ .Release.Name }}
     spec:
+{{- if .Values.controller.rbac.enabled }}
+      serviceAccountName: {{ template "fullname" . }}
+{{- end }}
```

Comment:
Should this also have a `default` service account? https://github.com/kubernetes/charts/blob/master/stable/kube2iam/templates/daemonset.yaml#L24